ocs_ci/deployment/deployment.py

```diff
@@ -134,11 +137,32 @@ def deploy_ocp(self, log_cli_level='DEBUG'):
         self.ocp_deployment = self.OCPDeployment()
         self.ocp_deployment.deploy_prereq()
         self.ocp_deployment.deploy(log_cli_level)
-        self.add_stage_cert()
         # logging the cluster UUID so that we can ask for it's telemetry data
         cluster_id = run_cmd("oc get clusterversion version -o jsonpath='{.spec.clusterID}'")
         logger.info(f"clusterID (UUID): {cluster_id}")
 
+    def post_ocp_deploy(self):
+        """
+        Function does post OCP deployment stuff we need to do.
+        """
+        # Workaround for #1777384 - enable container_use_cephfs on RHEL workers
```

Comment:
It might be worth to mention here also the RHSTOR-787 ticket. Without that it is slightly confusing, why it is needed, since the BZ is already CLOSED.

Reply:
done